app/v6/lossClaims/delete/DeleteLossClaimController.scala

```diff
@@ -43,24 +43,23 @@ class DeleteLossClaimController @Inject() (
   implicit val endpointLogContext: EndpointLogContext =
     EndpointLogContext(controllerName = "DeleteLossClaimController", endpointName = "Delete a Loss Claim")
 
-  def delete(nino: String, claimId: String): Action[AnyContent] =
+  def delete(nino: String, claimId: String, taxYear: String): Action[AnyContent] =
```

Comment:
Please update to the right name `taxYearClaimedFor`  for consistency and clarity, thanks. This is the naming convention used in list and amend order.